security/pkg/pki/testdata/multilevelpki/ecc-certs.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
```

Comment:
This is based off of certs.sh with EC params passed to openssl

Reply:
I can modify the `certs.sh` script to have a flag if warranted. The commands are quite different for RSA vs EC which is why I split them up.